src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -118,6 +99,35 @@ public SmartClinicalScopesMiddleware(RequestDelegate next)
                                 }
 
                                 fhirRequestContext.AccessControlContext.AllowedResourceActions.Add(new ScopeRestriction(resource, permittedDataActions, id));
+
+                                if (string.Equals("system", id, StringComparison.OrdinalIgnoreCase))
+                                {
+                                    skipFhirUserClaimForSystemScope = true;
+                                }
+                            }
+                        }
+                    }
+
+                    if (!skipFhirUserClaimForSystemScope)
```

Comment:
Just for legibility, I would suggest: 
* rename "skipFhirUserClaimForSystemScope" with "includeFhirUserClaimForSystemScope", inverting the usage of . Avoiding operations like "!skip" by "include". It'll make the code more logical and easier to read. 
* Adding comments explaining what it's the type of user claim being manipulated at this part of the code.

Reply:
Changed.